Common/src/main/resources/assets/toolleveling/lang/zh_cn.json

```diff
@@ -0,0 +1,119 @@
+    "enchantment.level.16": "16",
+    "enchantment.level.17": "17",
+    "enchantment.level.18": "18",
+    "enchantment.level.19": "19",
+    "enchantment.level.20": "20",
+    "enchantment.level.21": "21",
+    "enchantment.level.22": "22",
+    "enchantment.level.23": "23",
+    "enchantment.level.24": "24",
+    "enchantment.level.25": "25",
+    "enchantment.level.26": "26",
+    "enchantment.level.27": "27",
+    "enchantment.level.28": "28",
+    "enchantment.level.29": "29",
+    "enchantment.level.30": "30",
+    "enchantment.level.31": "31",
+    "enchantment.level.32": "32",
+    "enchantment.level.33": "33",
+    "enchantment.level.34": "34",
+    "enchantment.level.35": "35",
+    "enchantment.level.36": "36",
+    "enchantment.level.37": "37",
+    "enchantment.level.38": "38",
+    "enchantment.level.39": "39",
+    "enchantment.level.40": "40",
+    "enchantment.level.41": "41",
+    "enchantment.level.42": "42",
+    "enchantment.level.43": "43",
+    "enchantment.level.44": "44",
+    "enchantment.level.45": "45",
+    "enchantment.level.46": "46",
+    "enchantment.level.47": "47",
+    "enchantment.level.48": "48",
+    "enchantment.level.49": "49",
+    "enchantment.level.50": "50",
+    "enchantment.level.51": "51",
+    "enchantment.level.52": "52",
+    "enchantment.level.53": "53",
+    "enchantment.level.54": "54",
+    "enchantment.level.55": "55",
+    "enchantment.level.56": "56",
+    "enchantment.level.57": "57",
+    "enchantment.level.58": "58",
+    "enchantment.level.59": "59",
+    "enchantment.level.60": "60",
+    "enchantment.level.61": "61",
+    "enchantment.level.62": "62",
+    "enchantment.level.63": "63",
+    "enchantment.level.64": "64",
+    "enchantment.level.65": "65",
+    "enchantment.level.66": "66",
+    "enchantment.level.67": "67",
+    "enchantment.level.68": "68",
+    "enchantment.level.69": "69",
+    "enchantment.level.70": "70",
+    "enchantment.level.71": "71",
+    "enchantment.level.72": "72",
+    "enchantment.level.73": "73",
+    "enchantment.level.74": "74",
+    "enchantment.level.75": "75",
+    "enchantment.level.76": "76",
+    "enchantment.level.77": "77",
+    "enchantment.level.78": "78",
+    "enchantment.level.79": "79",
+    "enchantment.level.80": "80",
+    "enchantment.level.81": "81",
+    "enchantment.level.82": "82",
+    "enchantment.level.83": "83",
+    "enchantment.level.84": "84",
+    "enchantment.level.85": "85",
+    "enchantment.level.86": "86",
+    "enchantment.level.87": "87",
+    "enchantment.level.88": "88",
+    "enchantment.level.89": "89",
+    "enchantment.level.90": "90",
+    "enchantment.level.91": "91",
+    "enchantment.level.92": "92",
+    "enchantment.level.93": "93",
+    "enchantment.level.94": "94",
+    "enchantment.level.95": "95",
+    "enchantment.level.96": "96",
+    "enchantment.level.97": "97",
+    "enchantment.level.98": "98",
+    "enchantment.level.99": "99",
+    "enchantment.level.100": "100"
```

Comment:
Little tip, if you want to have the lines exactly like the english version, you can just erase them.  
The game will always use the english translation of everything if a translation is not found in the selected language.

Reply:
Yes, I thought about doing that, but I didn't for fear of accidents and for the sake of rigor. I'll delete them next time.